metal/classifier.py

```diff
@@ -195,6 +195,9 @@ def _create_checkpointer(self, checkpoint_config):
             model_class, **checkpoint_config, verbose=self.config["verbose"]
         )
 
+    def get_loss_fn(self):
```

Comment:
This needs an underscore. Also add a one-liner docstring describing the purpose of this and an error message with the exception that says it's unimplemented because it's an abstract class, not because we just haven't written it yet.

Reply:
Yep, will fix this.